src/Raven.Studio/typescript/components/pages/database/settings/clientConfiguration/ClientDatabaseConfiguration.tsx

```diff
@@ -0,0 +1,321 @@
+import React, { useMemo } from "react";
+import { Form, Col, Button, Card, Row, Spinner, Input } from "reactstrap";
+import { SubmitHandler, useForm, useWatch } from "react-hook-form";
+import { FormCheckbox, FormInput, FormSelect, FormSelectOption, FormSwitch } from "components/common/Form";
+import ReadBalanceBehavior = Raven.Client.Http.ReadBalanceBehavior;
+import { useServices } from "components/hooks/useServices";
+import { useAsync, useAsyncCallback } from "react-async-hook";
+import { LoadingView } from "components/common/LoadingView";
+import { useAccessManager } from "hooks/useAccessManager";
+import { LoadError } from "components/common/LoadError";
+import genUtils = require("common/generalUtils");
+import database = require("models/resources/database");
+import {
+    ClientConfigurationFormData,
+    clientConfigurationYupResolver,
+} from "components/common/clientConfiguration/ClientConfigurationValidation";
+import { Icon } from "components/common/Icon";
+import appUrl = require("common/appUrl");
+import ClientConfigurationUtils from "components/common/clientConfiguration/ClientConfigurationUtils";
+
+interface ClientDatabaseConfigurationProps {
+    db: database;
+}
+
+// TODO: show modal on exit intent if is dirty
+export default function ClientDatabaseConfiguration({ db }: ClientDatabaseConfigurationProps) {
+    const { manageServerService } = useServices();
+    const getClientConfigurationCallback = useAsyncCallback(manageServerService.getClientConfiguration);
```

Comment:
can we remove that?

Reply:
https://github.com/ravendb/ravendb/pull/16356#discussion_r1177798760